src/core/Pagination/PageInput/PageInput.baseStyles.tsx

```diff
@@ -0,0 +1,155 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { input, containerIEFocus, font } from '../../theme/reset';
+
+export const baseStyles = (theme: SuomifiTheme) => css`
+  ${font(theme)('bodyText')}
+
+  & .fi-page-input {
+    &_wrapper {
+      width: 100%;
+      min-width: 105px;
+      display: inline-block;
+
+      & .fi-page-input_label--visible {
+        margin-bottom: ${theme.spacing.xs};
+      }
+
+      & .fi-page-input_statusText--has-content {
+        margin-top: ${theme.spacing.xxs};
+      }
+    }
+
+    &_functionality-container {
+      position: relative;
+    }
+
+    &_input-element-container {
+      background-color: ${theme.colors.whiteBase};
+      color: ${theme.colors.blackBase};
+      ${containerIEFocus(theme)}
+
+      &:focus-within {
+        position: relative;
+        box-shadow: ${theme.shadows.actionElementBoxShadow};
+        &:after {
+          ${theme.focus.absoluteFocus}
+          top: -3px;
+          bottom: -3px;
+          right: -3px;
+          left: -3px;
+        }
+        > input:focus {
+          outline: none;
+        }
+      }
+      width: 100%;
+      height: 40px;
+      box-sizing: border-box;
+      border: 1px solid ${theme.colors.depthDark3};
+      border-radius: ${theme.radius.basic};
+    }
+
+    &_input {
+      ${input(theme)}
+      padding-top: ${theme.spacing.insetS};
+      padding-bottom: ${theme.spacing.insetS};
+      width: calc(100% - 24px);
+      min-width: 65px;
+      border: 0;
+      min-height: 36px;
+      margin-top: 1px;
+      margin-bottom: 1px;
+      &::placeholder {
+        font-style: italic;
+        color: ${theme.colors.depthDark2};
+        opacity: 1;
+      }
+      &::-ms-clear {
```

Comment:
I tested with Legacy Edge and seems ms-clear & ms-reveal are just for IE, so not necessary anymore.